docs/main/getting-started/environment-setup.md

```diff
@@ -20,7 +20,7 @@ Once you have installed Node, open your terminal of choice and type in the follo
 
 ```bash
 node --version
-# v18.3.0
+# v20.9.0
```

Comment:
this is just an example output, the version is a few lines before, where it says 16 it should now say 18

Reply:
Yeah I just set the example to the latest LTS version.  updated the min node version to be 18 now, thanks! I'll update the upgrade guide as well